cascadiaMain.go

```diff
@@ -137,6 +183,20 @@ func Cascadia(bi io.Reader, bw io.Writer, css string, piece MapStringString, del
 	return nil
 }
 
+func getInnerText(node *html.Node) string {
+	text := ""
+	if node.Type == html.TextNode {
+		text += node.Data
+	} else if node.Type == html.ElementNode && node.FirstChild != nil {
+		text += getInnerText(node.FirstChild)
+	}
+	if node.NextSibling != nil {
+		text += getInnerText(node.NextSibling)
+	}
```

Comment:
I don't quite understand the algorithm here. Please add sufficient test cases. E.g., for `<a>` tag or `<input>` tag, besides `TextNode`. Also, what about the case of more than one `NextSibling`?